stub-packages/utilities/CHANGELOG.json

```diff
@@ -0,0 +1,4677 @@
+{
+  "name": "@fluentui/utilities",
+  "entries": [
+    {
```

Comment:
I think we want to clear out the changelog for stub packages, right?

Reply:
yeah, i wasn't sure what the expectations were there